js_modules/dagit/packages/core/src/runs/LogsProvider.tsx

```diff
@@ -289,6 +289,7 @@ const LogsProviderWithQuery = (props: LogsProviderWithQueryProps) => {
   const {counts, cursor, nodes} = state;
 
   const {stopPolling, startPolling} = useQuery(RUN_LOGS_QUERY, {
+    fetchPolicy: 'cache-first', // Is this correct?
```

Comment:
Probably doesn't matter since variables are changing a lot and it will requery anyway? But seems like `cache-and-network` should be the correct policy here.

Reply:
I think you're right 👍